tmk_core/protocol/nrf/usbd.h

```diff
@@ -20,13 +20,13 @@ char cdc_acm_getc();
 
 #else
 int usbd_init(void) {return 0;}
-void usbd_enable(void) {}
-int usbd_process(void) {return 0;}
+int usbd_enable(void) {}
+void usbd_process(void) {return 0;}
```

Comment:
Please fix return-type error.
return 0 in usbd_enable(), and delete return statement in usbd_process()

Reply:
Oh! Yes, of course!